dashboard/app/models/pd/application/teacher1920_application.rb

```diff
@@ -352,6 +352,18 @@ def allow_sending_principal_email?
       true
     end
 
+    def allow_sending_application_reminder_email?
```

Comment:
can we also name this something more specific? "application reminder email" doesn't seem to describe what this email is about.

Reply:
I have gotten to `allow_sending_principal_approval_teacher_reminder_email?` :)